packages/app/src/pages/VideoSettings.tsx

```diff
@@ -76,6 +77,12 @@ export const VideoSettings = (): JSX.Element => {
     }
   };
 
+  useEffect(() => {
+    if (navType === 'POP') {
```

Comment:
Does 'POP' have en enum in react-router-dom we could use here instead of string?